Makefile

```diff
@@ -435,30 +435,28 @@ uninstall: uninstall_themis uninstall_soter
 # Themis distribution tarball
 #
 
-THEMIS_DIST_FILENAME = $(VERSION).tar.gz
+DIST_DIR = themis-$(VERSION)
```

Comment:
Excellent improving. But we use Debian-style naming through all our products, so I'd suggest to change `-` to `_`.